### PR TITLE
Add lesson on logging

### PR DESCRIPTION
**Pull request by ohler55**
Some changes were also made to wabur to cleanup logging.

### CODE REVIEW

File: lib/wab/impl/handler.rb [hunk 1]
```diff
@@ -52,7 +52,7 @@ def log_request(caller, path, query)
       end
 
       def log_request_with_body(caller, path, query, body)
-        @shell.logger.info("#{caller}(#{path.join('/')}#{query}, #{body.json})")
+        @shell.logger.info("#{caller} #{path.join('/')}#{query}\n#{body.json(@shell.indent)}")
```
Comment (ashmaroli) [inline lib/wab/impl/handler.rb]: with this we will have 3 lines that involve `.json(@shell.indent)`
would it be better if we extracted that to a private helper?
```ruby
prvate
def jsonify(data)
  data.json(@shell.indent)
end
```
```
jsonify(body)
jsonify(result)
jsonify(@shell.data(body))
```

Comment (ashmaroli) [reply lib/wab/impl/handler.rb]: perhaps better suited for another PR..

**Comment by ashmaroli** [comment]
oh wait a minute.. there is no mention regarding the `run-lesson-1` script in the tutorial..

File: tutorial/lesson-1/README.md [hunk 2]
```diff
@@ -240,3 +241,9 @@ pull in the wab UI JavaScript.
 ```
 
 Now when wabur is run the frame identifies the page as your blog.
+
+## Shortcut
+
+A script is available in the wabur gem that will execute all the steps in this
```
Comment (ashmaroli) [inline tutorial/lesson-1/README.md]: scripts won't be bundled into the gem.. they're hosted at the repo only.. (as of this writing..)

Comment (ohler55) [reply tutorial/lesson-1/README.md]: Easily changed although I'm not sure if that is best. Any preference?

Comment (ashmaroli) [reply tutorial/lesson-1/README.md]: well.. even if the script was bundled in the gem, the user would have to enter the correct path to the gem to locate the script via shell command..

Comment (ashmaroli) [reply tutorial/lesson-1/README.md]: The other alternative is to clone the repo, install dev-dependencies, run `build-export` and then the lesson script..

File: tutorial/lesson-2/README.md [hunk 3]
```diff
@@ -1,145 +1,248 @@
...
-            actions: {
-                createButton: 'entryCreate',
-                viewButton: 'entryView',
-                editButton: 'entryView',
-                deleteButton: 'entryList'
-            }
-        }
-    }
-}
+# WABuR Tutorial Lesson 3
```
Comment (ohaddahan) [inline tutorial/lesson-2/README.md]: Typo , this is lesson 2 , the new commit change it to lesson 3.

**Pull request merged**